families/burrow_evm/src/sawtooth_burrow_evm/client/client.go

```diff
@@ -47,23 +47,29 @@ func New(url string) *Client {
 	}
 }
 
-func (c *Client) Load(priv, init []byte, gas uint64) (*RespBody, error) {
-	address, err := PrivToAddr(priv)
+func (c *Client) Load(priv, init []byte, gas, nonce uint64) (*RespBody, error) {
```

Comment:
This is ugly - don't hide the type of gas by implicitly typing it the same is nonce (it has always been a terrible language feature in any language - it saves writing but adds cognitive load on reading).  Be explicit.